app/posts/manage-teacher-training-applications/2021-04-08-gathering-school-direct-contact-details-from-higher-education-institutions.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Gathering school direct contact details from higher education institutions
+description: Using a spreadsheet to help higher education institutions invite their school direct partners to use  Manage
+date: 2021-04-08
+---
+
+{% from "email/macro.njk" import appEmail %}
```

Comment:
Can you group this with the macro's usage later on, since you're only using it once?